pods/covid-spain-chart/communities.ts

```diff
@@ -0,0 +1,87 @@
+export const latLongCommunities = [
```

Comment:
Let's think about promoting this to common, is something we will reuse

Reply:
Sure